setup.py

```diff
@@ -684,7 +684,7 @@ def setup_package():
         'numpy', 'mako', 'cyarray', 'compyle>=0.8', 'Cython>=0.20',
         'setuptools>=42.0.0', 'pytools', 'Beaker'
     ]
-    tests_require = ['pytest>=3.0', 'h5py']
+    tests_require = ['pytest>=3.0', 'h5py', 'vtk']
```

Comment:
Is there a way to test this without using VTK?

Reply:
Yes, I would have to write my own xdmf reader but that would be really complicated. 
Also, if my xdmf reader reads it but somehow vtk's xdmf reader cannot, then I think it wouldn't be really useful.

Reply:
Is there a pure Python reader that you can use? I think `meshio` will do the job pretty well, and I think it may already be an optional requirement.

Reply:
I have tried `meshio` earlier because that would have been easier. I went with `vtk` since `meshio` had limited xdmf support. Other than that, xdmf.org has [something](https://gitlab.kitware.com/xdmf/xdmf) but that is also not pure python.